src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -443,6 +443,10 @@ public function getColumns($tableName)
                 $column->setPrecision($columnInfo['numeric_precision']);
             }
 
+            if ($column->getType() === static::PHINX_TYPE_BINARY && $column->getLimit() === 16) {
+                $column->setType(static::PHINX_TYPE_BINARYUUID);
+            }
```

Comment:
Is it possible to enter this if statement?

Reply:
Based on your findings it might not be necessary.

Reply:
Binary UUIDs in postgres would use the standard `UUID` type. Not a binary column.

Reply:
That means we could safely remove this?

Reply:
I believe so. You should always use the uuid type with postgres.

Reply:
Done